packages/iTwinUI-react/src/core/Table/Table.tsx

```diff
@@ -676,6 +676,20 @@ export const Table = <
     }
   }, [state, instance.filteredRows, onFilter]);
 
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
Remove eslint-disable and change `Columns<any>` to `Columns<T>`.

Reply:
**[DONE]**